content/adversary_village/_index.md

```diff
@@ -0,0 +1,61 @@
+
+For the first time in BSidesTLV history, we are excited to introduce the Adversary Village! This year's event will host a range of engaging activities designed to immerse attendees in the world of adversary simulation, emulation, malicious mindset, threat intelligence, and purple teaming. We invite you to join us in this groundbreaking initiative and be part of a collaborative effort to shape the future of cybersecurity.
+
+## Our Vision
+
+The Adversary Village is a dynamic and engaging community initiative that's all about Adversary simulation, Adversary Tactics, Life, Adversary Philosophy, Threat Actors Emulation, and Purple Teaming. Our goal is to provide a platform for professionals and enthusiasts to hone their skills and develop new tactics, techniques, and procedures to combat modern cyber threats. At BSidesTLV, we'll host technical talks, live demos, panel discussions, and hands-on activities to help you explore the latest tools and techniques in Adversary simulation and Purple teaming and to help you understand better the attackers' mindset and TTP, to create better defensive and offensive capabilities. Whether you're a seasoned cybersecurity pro or just starting out, we've got something for everyone. Join us at the Adversary Village, and let's make a tangible impact on the cybersecurity world together!
+
+## Call for Papers
+
+In line with our vision, we are looking for innovative and groundbreaking presentations that focus on the development and improvement of adversary tactics, TTPs, philosophy, skill set, and mindset. The Adversary Village will host technical talks, workshops, live demos, panel discussions, and other hands-on activities.
+We invite you to submit your proposals for talks at the Adversary Village during BSidesTLV on Thursday, June 29th, 2023. Our CFP is open from Tuesday, 30th March till May 1st, 2023.
+
+#### Topics we would like to hear about:
+
+We encourage outstanding submissions that demonstrate community-driven development and emphasize collaboration in areas such as:
+- Designing and executing realistic adversary simulations
+- Innovating and refining adversary emulation techniques
+- Enhancing purple teaming methodologies and tactics
+- Developing new tools and technologies for simulation and emulation
+- Crafting unique strategies to combat supply chain threats
+- Exploring the philosophy and mindset of adversaries
+- Actionable adversary intelligence 
+
+#### Available speaking slots include: - H3
+
+- Regular talk (25 minutes)
+- Deep-dive, technical talk (45 minutes, which should include a workshop or a live demo)
+- Lightning talk (10 minutes)
+- Panel (30 minutes, which should include a few members)
+
+Submit your proposal here:
+
+#### Call for Volunteers
+
+We're looking for volunteers to help us run the Adversary Village this year. If you're interested in joining our team, please reach out to us via email at ‫adversaryvillage@bsidestlv.com‬ or on Twitter @AdversaryVillageIL.
```

Comment:
can you replace the unicode chars?

Reply:
moved it into a link

Reply:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/639682/229348939-b18d227f-5936-4d50-8d2f-9c837deebfbb.png">
still seeing it